code/src/main/java/com/onlinecontacttracing/servlets/get-data/PeopleDataForPositiveUser.java

```diff
@@ -20,9 +53,40 @@ public void run() {
     // Initiate objects to store information
     PositiveUserContacts positiveUserContacts = new PositiveUserContacts(userId);
 
-    // TODO: add people api
+    try {
+      final NetHttpTransport httpTransport = GoogleNetHttpTransport.newTrustedTransport();
+
+      // Get user's calendar
+      PeopleService service = new PeopleService.Builder(httpTransport, JSON_FACTORY, credential)
+        .setApplicationName(APPLICATION_NAME)
+        .build();
+
+      // Request All connections.
+      ListConnectionsResponse response = service.people().connections()
+        .list("people/me")
+        .setPersonFields("names,emailAddresses")
+        .execute();
 
-    // Store data or replace old data with newer data.
-    // ofy.save().entity(positiveUserContacts).now();
+      // Store all connections available.
+      List<Person> connections = response.getConnections();
+      if (connections != null && connections.size() > 0) {
+        for (Person person : connections) {
+            List<Name> names = person.getNames();
+            if (names != null && names.size() > 0) {
+                positiveUserContacts.add(person.getNames().get(0).getDisplayName()
+                , person.getEmailAddresses().get(0).getDisplayName());
+            } else {
+                System.out.println("No names available for connection.");
```

Comment:
Let's remove these statements - I think they're just for debugging, right? Same below.